### DISCUSSION TITLE
Distroless Image Alpha or stable

### DISCUSSION THREAD

**Discussion by srinathrangaramanujam** [istio/istio#48669] [General]
Hey guys. Thank you for the great project. 
we are planning to use istio distroless as a part of security initiative. 
I just wanted to know if this is still in alpha or its stable to be deployed in production

**Comment by howardjohn** [comment]
Per https://istio.io/latest/docs/releases/feature-stages/#istio-features, it is beta. TBH it could probably be stable; I highly recommend it.

**Reply by srinathrangaramanujam** [reply]
Thats great to know. Thanks @howardjohn
